src/kili/services/label_data_parsing/json_response.py

```diff
@@ -66,23 +113,75 @@ def __init__(
                 job_payload=job_response,
             )
 
-    def __getitem__(self, job_name: str) -> "job_response_module.JobPayload":
-        """Returns the JobPayload object corresponding to the job name."""
-        if job_name not in self._json_data:
-            raise JobNotExistingError(job_name)
-        return self._json_data[job_name]
-
-    @property
-    def jobs(self) -> "ParsedJobs":
-        """Returns self."""
-        return self
-
-    def to_dict(self) -> Dict:
+    def to_dict(self) -> Dict[str, Dict]:
```

Comment:
comment for above that was in a previous PR:
in ParseOtherJobs, you test:
```
# a required parent job should have a non-empty json response
            if (
                not job_interface["isChild"]  # check if parent
                and job_interface["required"]  # check if required
                and "VIDEO" not in project_info["inputType"]  # can have empty frames
                and not job_response
            ):
                raise JobNotExistingError(job_name)
```
But we agreed with PM when I was working on json response validation that one can import a label from the API/SDK with empty response for required jobs. It is very useful for predictions labels for example when you want to upload predictions only for some jobs. But in the general case, the required feature only work when labeling on the platform.
It should be changed. Can be in another PR